### ISSUE TITLE
NVDA isn't reading items in a folder correctly.

### ISSUE THREAD

**Issue by audiogamer22** [nvaccess/nvda#17831]
### Steps to reproduce:
GO to dropbox using the run dialog, for example. e:\dropbox; make a change like copying a couple of folders into that folder and arrow up and down.

### Actual behavior:

I was unable to use the speech viewer as NVDA has slowed down to such an extent it's very hard to alt tab to another window,, it's also not reading out the windows as i alt tab.

NVDA just says the name of the folder and not the number, for example, brads folder, instead of brads folder 1 of 3.
### Expected behavior:

NVDA should speak the number after each folder.
### NVDA logs, crash dumps and other attachments:
Added
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2024.4.2
#### Windows version:
24h2
#### Name and version of other software in use when reproducing the issue:
dropbox v219.4.4463
#### Does the issue still occur after restarting your computer?
Sometimes.
#### If NVDA add-ons are disabled, is your problem still occurring?
Unknown as I don't want to restart NVDA just yet but I assume that once I do, it'll read with no issues.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes, although I haven't restarted my system yet.

If I do; i'm sure it will work for a while.

[nvda.log](https://github.com/user-attachments/files/19281302/nvda.log)

**Comment by SaschaCowley** [comment]
This also happens intermittently with OneDrive. I suspect it has something to do with NVDA being flooded with events, but have not investigated. Usually restarting NVDA fixes the issue.

**Comment by audiogamer22** [comment]
It does.

Is there a way to make sure NVDA devs see this or do we have to wait and see if it's fixed in a later update?

It's not so frequent that it makes NVDA unusable but it would be nice to know they're looking into it.

**Comment by SaschaCowley** [comment]
@audiogamer22 we have seen it, and will formally triage it soon

**Comment by audiogamer22** [comment]
@SaschaCowley Ah, I wasn't aware you were a dev. That's good to know.

I look forward to the issue being fixed.

I'm not sure how to reproduce this but I also noticed that when pressing alt tab, after this happened, NVDA slowed down to such an extent that if I didn't restart it, it would have made me have to go back to JAWS or another screen reader.

I think I mentioned that already but just thought I'd bring it up again. Are you guys aware of this issue and what might cause it?

**Comment by SaschaCowley** [comment]
Again, it sounds like NVDA is being flooded with events. This can cause NVDA to freeze up irrecoverably, so your best bet is usually to restart NVDA. This is something we're aware of and working on

**Comment by audiogamer22** [comment]
Alright, sounds good.

**Comment by seanbudd** [comment]
To debug this it would be useful to have a debug log.
Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.
From advanced preferences also enable "UIA", "MSAA" and "Events" in the debug logging list.

This is very likely a problem with the implementation of the application, as a starting point please submit feedback to dropbox to let them know about the problem.

**Comment by audiogamer22** [comment]
@seanbudd I'll leave it for now and if it continues in NVDAS next big release; I'll try and let them know but honestly,, I doubt there's much they can do.

This isn't just a dropbox issue as @SaschaCowley pointed out.

**Comment by seanbudd** [comment]
@audiogamer22 it's going to continue into the next release unless Dropbox fixes it, or we can find a way to suppress any events we find in the debug log file you provide. For handling the OneDrive case that Sascha raised, we'd need to report it to MS

**Comment by audiogamer22** [comment]
Well everything seams to be working fine for me now, I think I said it in my first post but it's hard to trigger it.

If you want to report it and you have the know how, go for it but I'm leaving it for now.

**Comment by seanbudd** [comment]
Closing as won't fix. We need a debug log as outlined in https://github.com/nvaccess/nvda/issues/17831#issuecomment-2756013150 and for this issue to be reported to dropbox. We will reopen this ticket when both of these actions have been done.

**Comment by audiogamer22** [comment]
You might as well remove that part about reopening it. Everything seams fine now.

as this seams more like an NVDA issue like @SaschaCowley  said, I won't be reporting it to dropbox, honestly I doubt they'd do anything anyway as I don't think they'd have an accessibility team who'd understand these types of issues.
